utils/src/ooinstall/install_transactions.py

```diff
@@ -41,6 +44,10 @@ def generate_inventory(hosts):
     if 'OO_INSTALL_STAGE_REGISTRY' in os.environ:
         base_inventory.write('oreg_url=registry.access.stage.redhat.com/openshift3/ose-${component}:${version}\n')
 
+    if any(host.hostname == installer_host for host in hosts):
```

Comment:
let's check all the other hostnames we know about like host.hostname and host.public_hostname.